inventory/byo/hosts.origin.example

```diff
@@ -113,6 +113,9 @@ openshift_release=v3.6
 # Downgrades are not supported and will error out. Be careful when upgrading docker from < 1.10 to > 1.10.
 # docker_version="1.12.1"
 
+# Specify whether to --enable-selinux on Docker (set false if using overlay/2 storage drivers before RHEL 7.4)
```

Comment:
@ashcrow ah, thanks. I overlooked that. Hmm, I see that used here:

```
roles/docker/tasks/systemcontainer_docker.yml:    l_docker_selinux_enabled: "{{ docker_selinux_enabled | default(true) | to_json }}"
roles/docker/templates/daemon.json:    "selinux-enabled": {{ l_docker_selinux_enabled | lower }},
roles/openshift_docker_facts/tasks/main.yml:      selinux_enabled: "{{ openshift_docker_selinux_enabled | default(None) }}"
```

Maybe I should be documenting `openshift_docker_selinux_enabled` instead?

I think I see the drift of your nit... when running Docker as a system container, we're not setting flags on the daemon, but options in a config json. So maybe like this instead:

```
# Specify whether to run Docker daemon with SELinux enabled in containers.
# (Needs to be false to use overlay/overlay2 storage drivers before RHEL 7.4)
# openshift_docker_selinux_enabled=True
```